python/api/validators.py

```diff
@@ -0,0 +1,18 @@
+import re
+
+from rest_framework.serializers import ValidationError
+
+CLUSTER_NAME_PATTERN = re.compile(
+    r"^[a-zA-Z]"  # starts with latin letter (upper/lower case)
+    r"[a-zA-Z0-9\-\. ]*?"  # latin letters (upper/lower case), digits, hyphens, dots, whitespaces
+    r"[a-zA-Z0-9]$"  # ends with latin letter (upper/lower case) or digit
+)  # as a result of this pattern min_length = 2
+
+
+class CharFieldMatchValidator:
```

Comment:
Use pls `RegexValidator` for parent class, for example https://github.com/arenadata/adcm/pull/2015/files#diff-affe7e9dda5e6a57a496094f26a4c70ae4f1f7603a14b5c3f1a6199221ce19c1R48

Reply:
done